prusti/tests/verify/todo/set-contains-insert-remove.rs

```diff
@@ -0,0 +1,40 @@
+#![feature(rustc_private)]
+
+extern crate prusti_contracts;
+
+use std::collections::HashSet;
+use std::hash::Hash;
+use std::cmp::Eq;
+
+struct SetWrapper<T: Hash + Eq> {
+    set: HashSet<T>
+}
+
+impl<T: Hash + Eq> SetWrapper<T> {
+    #[pure]
+    #[trusted]
+    pub fn contains(&self, value: &T) -> bool {
+        self.set.contains(value)
+    }
+
+    #[ensures="self.contains(&value)"]
+    pub fn insert(&mut self, value: T) {
+        self.set.insert(value);
+    }
+
+    #[ensures="!self.contains(&value)"]
+    pub fn remove(&mut self, value: &T) -> bool {
+        self.set.remove(value)
+    }
+}
+
+fn main() {
+    let mut set = SetWrapper { set: HashSet::new() };
+    set.insert(1);
+    set.insert(2);
+    set.insert(3);
+    set.remove(&3);
+    set.remove(&2);
+    set.remove(&1);
+    set.remove(&1);
+}
```

Comment:
The error is the following, and we just did not implement this case yet.

> not yet implemented: Inhale the predicate rooted at dst_field.

Could you please remove this program from the PR and open an issue?